lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = false; // replace null with your response
```

Comment:
yes. Think of it as the "children scope" have access to "parent's" scope, but not vice versa.